source/administration/config-settings.rst

```diff
@@ -2237,6 +2237,30 @@ There are a number of settings customizable in ``config.json`` unavailable in th
 Service Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Cluster Log Timeout
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This setting defines the frequency of cluster request time logging for [performance monitoring](https://docs.mattermost.com/deployment/metrics.html), measured in milliseconds.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ClusterLogTimeoutMilliseconds": "2000"`` with whole number input.                                                       |
```

Comment:
If it's whole number input, then it should be ``2000`` only, without the quotes.